plugins/modules/helm.py

```diff
@@ -105,6 +105,31 @@
       - Run C(helm repo update) before the operation. Can be run as part of the package installation or as a separate step (see Examples).
     default: false
     type: bool
+  set_values:
```

Comment:
What happens when a user specifies both `set_values` and `release_values`/`values_files`? The helm docs seem to suggest you can only use one or the other. If that's the case, we will need to make sure we're appropriately addressing that restriction here.

Reply:
both can be used, adding tests cases to validate that